src/pages/messages/components/MessageList.tsx

```diff
@@ -28,31 +28,39 @@ export const MessageList = ({ ticket }: MessageListProps) => {
           gap: '0.25rem',
         }}>
         {messages.map((message) => (
-          <MessageItem key={message.identifier} message={message} ticketType={ticket.type} />
+          <MessageItem
+            key={message.identifier}
+            text={message.text}
```

Comment:
hva skjer om meldingen er veldig lang? burde nok ha noe trucation i MessageItem, som vi har på resultatsøk feks

Reply:
har makslengde på 160 tegn, så lar denne være for nå tror jeg. Vi burde egentlig hatt en håndtering på det om det legges inn fra andre steder da, men ikke topp-pri nå sånn jeg ser det